internal/mapping.go

```diff
@@ -164,3 +167,60 @@ func GetSpannerKeyName(srcKeyName string, schemaForeignKeys map[string]bool) str
 	schemaForeignKeys[spKeyName] = true
 	return spKeyName
 }
+
+func findTableLowerCase(referTable string, conv *Conv) string {
```

Comment:
Suggest a few minor tweaks:
a) renaming to ResolveTableRef (more declarative description, and this might be useful outside of this module)
b) do the cheap lookup first -- the mismatched capitalization is not the typical case
c) some minor go style changes

// ResolveTableRef returns the resolved table name corresponding to tableRef.
// Spanner requires that DDL references match the case of the referenced object,
// but this is not so for many source databases.
func ResolveTableRef(tableRef string, conv *Conv) (string, error) {
         if _, ok := conv.SpSchema[tableRef]; ok {
		return tableRef, nil
         }
	// Do case-insensitive search for tableRef.
	tableRef = strings.ToLower(tableRef)
	for t, _ := range conv.SpSchema {
		if strings.ToLower(table) == referTable {
			return t
		}
	}
	return referTable
}

Reply:
Thanks nevin, this tweaks works, also updated function references everywhere applicable.